core/util/trt_util.cpp

```diff
@@ -208,56 +160,37 @@ nvinfer1::Dims unpadDims(const nvinfer1::Dims& d) {
   return dims;
 }
 
-nvinfer1::Dims unsqueezeDims(const nvinfer1::Dims& d, int pos) {
+nvinfer1::Dims unsqueezeDims(const nvinfer1::Dims& d, int pos, int val, bool use_zeros) {
```

Comment:
Why do we need both val and use_zeros?

Reply:
use _zeros controls whether we should be using 0 instead of -1 on the shape. I have found that for convolutions, we do need to use 0s when there are more than 1 dynamic dimensions. Note it also sets zeroIsPlaceholder attribute for the shuffle.  Some pooling cases failed with that setting though, therefore I made it optional.